packages/type-utils/tests/getTypeName.test.ts

```diff
@@ -30,7 +32,7 @@ describe('getTypeName', () => {
 
   describe('returns primitive type', () => {
     it.each([
-      ['type Test = string;', 'string'],
+      // ['type Test = string;', 'string'],
```

Comment:
should this have the reenable todo as well?

Reply:
Oh, I meant to un-comment this...

```suggestion
      ['type Test = string;', 'string'],
```